File: server/php/src/Infrastructure/Mappers/PaymentDetails/AcquirerInformationMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\AcquirerInformation as AcquirerInformationDto;
use OnlinePayments\Sdk\Domain\AcquirerInformation as AcquirerInformationSdk;

final class AcquirerInformationMapper
{
    public static function mapFromSdkResponse(?AcquirerInformationSdk $response): ?AcquirerInformationDto
    {
        if ($response === null) {
            return null;
        }

        $dto = new AcquirerInformationDto();
        $dto->acquirerSelectionInformation = AcquirerSelectionInformationMapper::mapFromSdkResponse($response?->acquirerSelectionInformation ?? null);
        $dto->name = $response?->name ?? null;

        return $dto;
    }
}

File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/AcquirerInformationMapper.cs
using AcquirerInformationSdk = OnlinePayments.Sdk.Domain.AcquirerInformation;
using AcquirerInformationDto = Business.Domain.Payments.PaymentDetails.AcquirerInformation;

namespace Infrastructure.Mappers.PaymentDetails;

public static class AcquirerInformationMapper
{
    public static AcquirerInformationDto Map(AcquirerInformationSdk? response)
    {
        return new()
        {
            AcquirerSelectionInformation = AcquirerSelectionInformationMapper.Map(response?.AcquirerSelectionInformation),
            Name = response?.Name
        };
    }
}